src/Fmt_ast.ml

```diff
@@ -1969,12 +1969,14 @@ and fmt_expression c ?(box = true) ?epi ?eol ?parens ?ext
                   (sugar_sequence c width xexp)
                   ( match c.conf.sequence_style with
                   | `Separator -> " ;@;<1000 0>"
-                  | `Terminator -> ";@;<1000 0>" )
+                  | `Terminator -> ";@;<1000 0>"
+                  | `Breaker -> ";@;<1000 0>" )
                   (fun grp ->
                     list grp
                       ( match c.conf.sequence_style with
                       | `Separator -> " ;@ "
-                      | `Terminator -> ";@ " )
+                      | `Terminator -> ";@ "
+                      | `Breaker -> ";@;<1000 0>" )
```

Comment:
The implementation of breaker is a variation on terminator. It isn't clear to me why this option is combined into sequence_style, as all 4 combinations make sense don't they?